src/test/scala/com/wavesplatform/UtxPoolSpecification.scala

```diff
@@ -50,6 +51,22 @@ class UtxPoolSpecification extends FreeSpec
   } yield TransferTransaction.create(None, sender, recipient, amount, time.getTimestamp(), None, fee, Array.empty[Byte]).right.get)
     .label("transferTransaction")
 
+  private def onlyOneValidFromManyTransfers = (for {
+    (sender, senderBalance, state) <- stateGen
+    recipient <- accountGen
+    fee <- chooseNum(1, (senderBalance * 0.01).toLong)
+    offset <- chooseNum(1000L, 2000L)
+  } yield {
+    val time = new TestTime(System.currentTimeMillis())
```

Comment:
Looks like `TestTime` is always initialized to current time in this test, so it's probably a good idea to change default constructor. Please also check other tests where `TestTime` is used.